packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1131,14 +1131,23 @@ export class AccountHandler {
       return response;
     };
 
-    await checkCustomsAndLoadAccount();
-    await checkEmailAndPassword();
-    await checkSecurityHistory();
-    await checkTotpToken();
-    await createSessionToken();
-    await sendSigninNotifications();
-    await createKeyFetchToken();
-    return await createResponse();
+    try {
+      await checkCustomsAndLoadAccount();
+      await checkEmailAndPassword();
+      await checkSecurityHistory();
+      await checkTotpToken();
+      await createSessionToken();
+      await sendSigninNotifications();
+      await createKeyFetchToken();
+      return await createResponse();
+    } catch (e) {
```

Comment:
I did consider it but decided against it because I didn't want to add something for a specific failure to code that's executed for every response.  Maybe when we are on the third one of these type of pings, we'll create something that can be called from the error handler.

Reply:
Gotcha, it seemed like having a try/catch around the entire code block here might imply a better place to handle the error. Lets revisit if we add more.

Reply:
> having a try/catch around the entire code block 

Oh yeah, I hated it  🥲